scripts/create_pr.sh

```diff
@@ -0,0 +1,7 @@
+BRANCH_NAME=$1
```

Comment:
I'd like to propose renaming this script to "commit_unstaged_changes" or something like this, what do you think, does it make sense?

Reply:
Yep, that one is better, fixed: 0be5b37b57ae76c182a42f787ec920d9685e56e2